service-registry/src/main/java/io/servicecomb/serviceregistry/api/registry/Microservice.java

```diff
@@ -34,6 +35,11 @@
 public class Microservice {
   private String serviceId;
 
+  private Framework framework;
+
+  @JsonProperty(value = "registerBy")
+  private String registeredBy;
```

Comment:
use the original name
i remember wrote comment, class Microservice not only describe self, but also describe others

why did not do any reply?

Reply:
done